integration_tests/assets/etc/wazo-calld/conf.d/20-real-asterisk.yml

```diff
@@ -2,3 +2,4 @@ ari:
   connection:
     username: xivo
     password: xivo
+max_meeting_participants: 4
```

Comment:
Is that enforced somewhere else or am I missing something? 
Should that be in the wazo-confd API and defined for each meeting? I'm not sure I understand

Reply:
In `config.py` it is set to 25 which is the current "advertised" limit. I made it a configuration parameter to be able to test the limit without using 26 calls and to be able to change that limit more easily if/when we decide that 25 is not the limit anymore.